src/Data/Queries.re

```diff
@@ -0,0 +1,84 @@
+let allConfessionsQuery = "query {
```

Comment:
similar comment to the decode one. it's a bit weird to me to group this in this way; grouping them into one "Confession" or "comment" folder would make the most sense to me:

Ex:
Data
>Comment
>>decode
>>query
>>type

>Confession
>>decode
>>query
>>type